jerry-core/ecma/base/ecma-globals.h

```diff
@@ -741,18 +741,12 @@ typedef uintptr_t ecma_external_pointer_t;
   */
 typedef struct
 {
-  uint16_t status_flags;            /**< various status flags */
+  uint16_t size;                    /**< real size >> MEM_ALIGNMENT_LOG */
+  uint16_t refs;                    /**< reference counter for the byte code */
+  uint16_t status_flags;            /**< various status flags. */
```

Comment:
The flags contains whether the byte code is function or regexp, so it is still important. The other flag bits depends on this.